tst/Help.Tests.ps1

```diff
@@ -32,8 +32,12 @@ Describe "Testing module help" -Tag 'Help' -ForEach @{ exportedFunctions = $expo
         It 'Has link sections' {
             $help.psobject.properties.name -match 'relatedLinks' | Should -Not -BeNullOrEmpty -Because 'all exported functions should at least have link to online version as first Uri'
 
+            $functionName = $_.Name
+            $alias = Get-Alias -Name Should* | Where-Object { $_.Definition -eq $functionName }
+            $helpName = if ($alias) { $alias.Name } else { $help.Name }
+
             $firstUri = $help.relatedLinks.navigationLink | Where-Object uri | Select-Object -First 1 -ExpandProperty uri
-            $firstUri | Should -Be "https://pester.dev/docs/commands/$($help.Name)" -Because 'first uri-link should be to online version of this help topic'
+            $firstUri | Should -Be "https://pester.dev/docs/commands/$helpName" -Because 'first uri-link should be to online version of this help topic'
```

Comment:
`Get-Help Should-Be` will still show `Assert-Be`. We'll also need to modify `generate-command-reference.ps1` in docs-repo to rename files and title to match alias.

Reply:
https://github.com/pester/Pester/issues/2476